packages/jmespath/src/errors.ts

```diff
@@ -108,13 +108,6 @@ class ParseError extends JMESPathError {
  * Error thrown when an incomplete expression is encountered during parsing.
  */
 class IncompleteExpressionError extends ParseError {
-  /**
-   * Expression that was being parsed when the error occurred.
-   *
-   * Can be set by whatever catches the error.
-   */
-  public expression?: string;
-
   public constructor(options: {
```

Comment:
Is this related to the PR?

Reply:
Changing the `target` in `tsconfig.json` to Node.js 18 caused this issue to pop up.

This error extends another base error that has the same `expression` property, which was redeclared here, thus causing the error.

I should've added a comment in the PR to explain this, my bad.